credentials/gs1-8110-coupon/credential.json

```diff
@@ -0,0 +1,33 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://contexts.vcplayground.org/examples/gs1-8110-coupon/v1.json"
+  ],
+  "type": ["VerifiableCredential", "GS18110CouponCredential"],
+  "issuer": {
+    "id": "did:key:z6MkrHKzgsahxBLyNAbLQyB1pcWNYC9GmywiWPgkrvntAZcj",
+    "name": "Big Retail Store",
+    "url": "https://example.com",
+    "image": "https://vcplayground.org/examples/gs1-8110-coupon/logo.png"
+  },
+  "name": "Digital coupon: Buy one, get one free 16 oz coffees.",
+  "issuanceDate": "2023-06-04T00:00:00Z",
+  "image": "https://vcplayground.org/examples/gs1-8110-coupon/image.png",
+  "credentialSubject": {
+    "id": "did:key:123",
+    "clippedCoupon": {
+      "type": "CouponOffer",
+      "upc": "5-12345-000-10-7",
```

Comment:
Term not defined.

Reply:
Fixed. That should have been `gtin` anyhow--which subsumed the older UPC format sometime ago (though "UPC" is still used to reference GTIN-12's apparently...hence my confusion).